Update to 1.19.3

ooNoone opened the pull request:
can u plese update to 1.19.3 please?, i cant use it in 1.19.3

CyberFlameGO commented:
Hi! Unfortunately 1.19.3 support isn't finalised (the latest revision of the repo is unusable) - my branch https://github.com/CyberFlameGO/viewmodel-changer/tree/fix/resolve-1.19.3-compilation-errors has my latest attempt but I'm unable to figure it out.

CyberFlameGO commented:
@CoderIL This has now been completed \:) [Here](https://github.com/CyberFlameGO/viewmodel-changer/releases/tag/v1.4)'s the link if you're still interested!

renovate[bot] committed:
Update gradle/gradle-build-action digest to 6778644

renovate[bot] committed:
Update plugin fabric-loom to v1.2.2

renovate[bot] committed:
Update dependency gradle to v8.1.1

renovate[bot] committed:
Update gradle/gradle-build-action digest to 9cf9903

renovate[bot] committed:
Update plugin fabric-loom to v1.2.5

renovate[bot] committed:
Update gradle/gradle-build-action digest to 55d241b

renovate[bot] committed:
Update gradle/gradle-build-action digest to 12681a2

renovate[bot] committed:
Update plugin fabric-loom to v1.2.6

renovate[bot] committed:
Update gradle/gradle-build-action digest to d23c38f

renovate[bot] committed:
Update plugin fabric-loom to v1.2.7

renovate[bot] committed:
Update gradle/gradle-build-action digest to 680037c

CyberFlameGO committed:
fix codeql alerts (#40)

* Update FloatSetting.java

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

* Update BooleanSetting.java

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

* Update ViewmodelScreen.java

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

* apply jetbrains' inspection stuff

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

---------

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
change dependabot

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
replace deprecated class with static method call

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
add code scanning tools (#42)

* Create crda.yml

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

* Create devskim.yml (#41)

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

---------

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
allow telemetry

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
nvm too much hassle

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

dependabot[bot] committed:
Bump gradle/gradle-build-action (#44)

dependabot[bot] committed:
Bump gradle/gradle-build-action from 1b2daf58338ede8c62b0fa54355d1f515f659ad8 to 8e02c8272aba4650563824b7012f8a86158901c6 (#46)

Bumps [gradle/gradle-build-action](https://github.com/gradle/gradle-build-action) from 1b2daf58338ede8c62b0fa54355d1f515f659ad8 to 8e02c8272aba4650563824b7012f8a86158901c6.
- [Release notes](https://github.com/gradle/gradle-build-action/releases)
- [Commits](https://github.com/gradle/gradle-build-action/compare/1b2daf58338ede8c62b0fa54355d1f515f659ad8...8e02c8272aba4650563824b7012f8a86158901c6)

---
updated-dependencies:
- dependency-name: gradle/gradle-build-action
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

renovate[bot] committed:
Update plugin fabric-loom to v1.3.2

renovate[bot] committed:
Update gradle/gradle-build-action digest to bde650d

renovate[bot] committed:
Update gradle/wrapper-validation-action digest to 064a97f

renovate[bot] committed:
Update dependency gradle to v8.2

renovate[bot] committed:
Update plugin fabric-loom to v1.3.4 (#52)

renovate[bot] committed:
Update plugin fabric-loom to v1.3.7

renovate[bot] committed:
Update plugin fabric-loom to v1.3.8

renovate[bot] committed:
Update gradle/gradle-build-action digest to 7a67f39

renovate[bot] committed:
Update dependency gradle to v8.2.1

renovate[bot] committed:
Update gradle/gradle-build-action digest to ce999ba

renovate[bot] committed:
Update gradle/gradle-build-action digest to 3c11eee

renovate[bot] committed:
Update mikepenz/gradle-dependency-submission action to v0.9.0

renovate[bot] committed:
Update gradle/gradle-build-action digest to a0fdbb0

renovate[bot] committed:
Update gradle/gradle-build-action digest to 2dbad1e (#64)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

CyberFlameGO committed:
calm down dependabot (#65)

* calm down dependabot

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

* calm down dependabot

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

---------

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
1.20 compatibility if it'll compile?

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
improvement(compatibility)!: add 1.20 support

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
improvement(compatibility)!: add 1.20.1 support + add icon

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
bump mod version to 1.7 for 1.20.1

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

renovate[bot] committed:
Update plugin fabric-loom to v1.3.8 (#68)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

CyberFlameGO committed:
refactor(groupid)!: change group and bump fabric version

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
bump version

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
workflow updates (#70)

* Update codeql-analysis.yml

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

* Create qodana.yml

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

---------

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
forgot to change (copied this from another of my repos)

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
clean up readme

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
Merge branch 'master' of https://github.com/CyberFlameGO/viewmodel-changer

CyberFlameGO committed:
docs(comment): yeet out useless todo

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

dependabot[bot] committed:
Bump JetBrains/qodana-action (#72)

CyberFlameGO committed:
preparation for 1.9 - adds sentry for error tracking

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
add auth token to workflows + use semeru (#74)

* Update release.yml

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

* Update codeql-analysis.yml

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

* Update build.yml

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

* Update release.yml

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

---------

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
fix branch

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
seems like there is a difference here contrary to ide tooltip

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
fix

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
oops forgor closey brace

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
add inspection thing to repo so i can pull on other laptop

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
this perhaps fixes things?

renovate[bot] committed:
chore(deps): update gradle/wrapper-validation-action digest to 56b90f2 (#75)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

CyberFlameGO committed:
okay actually reading the error helped

Signed-off-by: Aaron Lowe <cyberflameu@gmail.com>

CyberFlameGO committed:
[ci skip] update readme

CyberFlameGO committed:
[ci skip] clarify why 1.20 has a specific note in Vers. supported.

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
testing on master branch haha

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
testing on master branch haha v2

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
cleanup cleanup

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
refactors

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
cleanup!

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
1.9 release commit

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

renovate[bot] committed:
Update dependency io.sentry:sentry to v6.26.0 (#77)

renovate[bot] committed:
Update step-security/harden-runner digest to 9768986 (#76)

CyberFlameGO committed:
comment out sentry task stuff and yeet out sentry init code

Signed-off-by: Aaron Lowe <cyberflameu@gmail.com>

CyberFlameGO committed:
dependapendapendabot

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
dependerbot

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
whoops

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
oh.

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
update token name

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
um?

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
minor code changes

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
minor code changes part 2

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
kindof change stopwatch..?

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
seal settings interface

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
disable faily qodana grr

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
bump patch version

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
further code cleanup - next release likely to be cleanup release

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
mod version 1.9.2

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
update latest tag for 1.20.1

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
remove untested note for v1.9.2

CyberFlameGO committed:
add strictfp to getV

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
fix(mixin)!: adjust mixin of held item renderer to overwrite rather than inject

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
Merge branch 'master' of https://github.com/CyberFlameGO/viewmodel-changer

CyberFlameGO committed:
suppress redundant strictfp - serves as clarificative keyword

CyberFlameGO committed:
refactor: change getBooleanValue to isTrue, reformatting and removal of some casts

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
[no ci] bump loader version!

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

renovate[bot] committed:
Update dependency ubuntu to v22 (#83)

renovate[bot] committed:
Update plugin fabric-loom to v1.3.9

CyberFlameGO committed:
bump vers + formatting up

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

renovate[bot] committed:
Update dependency gradle to v8.3 (#85)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump actions/checkout from 3 to 4 (#88)

Bumps [actions/checkout](https://github.com/actions/checkout) from 3 to 4.
- [Release notes](https://github.com/actions/checkout/releases)
- [Changelog](https://github.com/actions/checkout/blob/main/CHANGELOG.md)
- [Commits](https://github.com/actions/checkout/compare/v3...v4)

---
updated-dependencies:
- dependency-name: actions/checkout
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

CyberFlameGO committed:
remove unnecessaries

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

renovate[bot] committed:
Update gradle/wrapper-validation-action digest to 342dbeb

renovate[bot] committed:
Update plugin fabric-loom to v1.4.1

renovate[bot] committed:
Update dependency gradle to v8.4

renovate[bot] committed:
Update plugin fabric-loom to v1.4.3

renovate[bot] committed:
Update plugin fabric-loom to v1.4.4

dependabot[bot] committed:
Bump mikepenz/gradle-dependency-submission from 0.9.0 to 0.9.1 (#95)

Bumps [mikepenz/gradle-dependency-submission](https://github.com/mikepenz/gradle-dependency-submission) from 0.9.0 to 0.9.1.
- [Release notes](https://github.com/mikepenz/gradle-dependency-submission/releases)
- [Commits](https://github.com/mikepenz/gradle-dependency-submission/compare/v0.9.0...v0.9.1)

---
updated-dependencies:
- dependency-name: mikepenz/gradle-dependency-submission
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

renovate[bot] committed:
Update dependency gradle to v8.5

renovate[bot] committed:
Update plugin fabric-loom to v1.4.5

CyberFlameGO committed:
test local threat model

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

dependabot[bot] committed:
Bump github/codeql-action from 2 to 3

Bumps [github/codeql-action](https://github.com/github/codeql-action) from 2 to 3.
- [Release notes](https://github.com/github/codeql-action/releases)
- [Changelog](https://github.com/github/codeql-action/blob/main/CHANGELOG.md)
- [Commits](https://github.com/github/codeql-action/compare/v2...v3)

---
updated-dependencies:
- dependency-name: github/codeql-action
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

CyberFlameGO committed:
add 1.20.2 (#106)

* Update gradle.properties

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

* Update fabric.mod.json

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

---------

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
1.20.2-specific changes (#107)

* 1.20.2-specific changes

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

* Update ViewmodelGuiObj.java

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

* use larger value of increment

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

---------

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
classic semicolon missing 

manual edit moment

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

dependabot[bot] committed:
Bump actions/upload-artifact from 3 to 4 (#104)

Bumps [actions/upload-artifact](https://github.com/actions/upload-artifact) from 3 to 4.
- [Release notes](https://github.com/actions/upload-artifact/releases)
- [Commits](https://github.com/actions/upload-artifact/compare/v3...v4)

---
updated-dependencies:
- dependency-name: actions/upload-artifact
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump actions/setup-java from 3 to 4 (#98)

* Bump actions/setup-java from 3 to 4

Bumps [actions/setup-java](https://github.com/actions/setup-java) from 3 to 4.
- [Release notes](https://github.com/actions/setup-java/releases)
- [Commits](https://github.com/actions/setup-java/compare/v3...v4)

---
updated-dependencies:
- dependency-name: actions/setup-java
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

* Update build.yml

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

* Update dependency-submission.yml

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

* Update release.yml

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

* Update build.yml

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

---------

Signed-off-by: dependabot[bot] <support@github.com>
Signed-off-by: CyberFlame <cyberflameu@gmail.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
partial revert (#108)

* revert

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

* Update release.yml

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

---------

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

renovate[bot] committed:
Update plugin fabric-loom to v1.5.3

renovate[bot] committed:
Update plugin fabric-loom to v1.5.4

renovate[bot] committed:
Update mikepenz/gradle-dependency-submission action to v0.9.2

renovate[bot] committed:
Update plugin fabric-loom to v1.5.5

renovate[bot] committed:
Update plugin fabric-loom to v1.5.6

renovate[bot] committed:
Update plugin fabric-loom to v1.5.7

renovate[bot] committed:
Update gradle/wrapper-validation-action digest to 1ff5a18

renovate[bot] committed:
Update gradle/wrapper-validation-action digest to 83cf5fd

renovate[bot] committed:
Update dependency gradle to v8.6

renovate[bot] committed:
Update gradle/wrapper-validation-action digest to 85cde3f

renovate[bot] committed:
Update gradle/wrapper-validation-action digest to 63d15e7

renovate[bot] committed:
Update plugin fabric-loom to v1.5.8

renovate[bot] committed:
Update plugin fabric-loom to v1.6.3

renovate[bot] committed:
Update gradle/wrapper-validation-action digest to b231772

renovate[bot] committed:
Update dependency gradle to v8.7 (#135)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
Update plugin fabric-loom to v1.6.5

renovate[bot] committed:
Update plugin fabric-loom to v1.6.6

renovate[bot] committed:
Update gradle/wrapper-validation-action digest to b5418f5

renovate[bot] committed:
Update gradle/wrapper-validation-action digest to 460a3ca

renovate[bot] committed:
Update plugin fabric-loom to v1.6.9

renovate[bot] committed:
Update plugin fabric-loom to v1.6.10

renovate[bot] committed:
Update gradle/wrapper-validation-action digest to 5188e9b

renovate[bot] committed:
Update plugin fabric-loom to v1.6.11

renovate[bot] committed:
Update gradle/wrapper-validation-action digest to 216d1ad

renovate[bot] committed:
Update dependency gradle to v8.8

renovate[bot] committed:
Update plugin fabric-loom to v1.6.12

renovate[bot] committed:
Update plugin fabric-loom to v1.7.1

renovate[bot] committed:
Update gradle/wrapper-validation-action digest to 9ba54b6

renovate[bot] committed:
Update gradle/wrapper-validation-action digest to 8842585

renovate[bot] committed:
Update plugin fabric-loom to v1.7.2

renovate[bot] committed:
Update dependency gradle to v8.9

CyberFlameGO committed:
add 1.21 support (#156)

* add 1.21 support

Signed-off-by: Aaron Lowe <cyberflameu@gmail.com>

* Bump gradle/gradle-build-action from 2 to 3 (#120)

Bumps [gradle/gradle-build-action](https://github.com/gradle/gradle-build-action) from 2 to 3.
- [Release notes](https://github.com/gradle/gradle-build-action/releases)
- [Commits](https://github.com/gradle/gradle-build-action/compare/v2...v3)

---
updated-dependencies:
- dependency-name: gradle/gradle-build-action
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

---------

Signed-off-by: Aaron Lowe <cyberflameu@gmail.com>
Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

CyberFlameGO committed:
fix stuff (#157)

* Update build.yml

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

* Update release.yml

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

---------

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
fix 1.21 not building

Signed-off-by: Aaron Lowe <cyberflameu@gmail.com>

CyberFlameGO committed:
update workflows

Signed-off-by: Aaron Lowe <cyberflameu@gmail.com>

CyberFlameGO committed:
whoops - forgot w

Signed-off-by: Aaron Lowe <cyberflameu@gmail.com>

dependabot[bot] committed:
Bump actions/dependency-review-action from 3 to 4 (#115)

Bumps [actions/dependency-review-action](https://github.com/actions/dependency-review-action) from 3 to 4.
- [Release notes](https://github.com/actions/dependency-review-action/releases)
- [Commits](https://github.com/actions/dependency-review-action/compare/v3...v4)

---
updated-dependencies:
- dependency-name: actions/dependency-review-action
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

CyberFlameGO committed:
fix typo

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
bump version 

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

renovate[bot] committed:
Update plugin fabric-loom to v1.7.3

dependabot[bot] committed:
Bump gradle/actions from 3 to 4 (#162)

Bumps [gradle/actions](https://github.com/gradle/actions) from 3 to 4.
- [Release notes](https://github.com/gradle/actions/releases)
- [Commits](https://github.com/gradle/actions/compare/v3...v4)

---
updated-dependencies:
- dependency-name: gradle/actions
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

renovate[bot] committed:
Update gradle/actions action to v4 (#161)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

CyberFlameGO committed:
1.21.1 (#163)

* fix silly oversight

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

* Update gradle.properties

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

* Update fabric.mod.json

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

* Update viewmodel.mixins.json

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

---------

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
add modrinth and info

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

renovate[bot] committed:
Update dependency gradle to v8.10

renovate[bot] committed:
Update plugin fabric-loom to v1.7.4

renovate[bot] committed:
Update dependency gradle to v8.10.1

renovate[bot] committed:
Update dependency gradle to v8.10.2 (#169)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

CyberFlameGO committed:
possible bugfix

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
testing on master

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
test 2?

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
revert to 1.21.1 (#170)

* revert commit 1

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

* half-revert

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

---------

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

renovate[bot] committed:
Update plugin fabric-loom to v1.8.2

renovate[bot] committed:
Update plugin fabric-loom to v1.8.5

renovate[bot] committed:
Update plugin fabric-loom to v1.8.6

renovate[bot] committed:
Update plugin fabric-loom to v1.8.8

renovate[bot] committed:
Update plugin fabric-loom to v1.8.9

CyberFlameGO committed:
Add keybind configuration (#176)

* add keybind api

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

* implement feature

Signed-off-by: Aaron Lowe <cyberflameu@gmail.com>

* add translations, bump version

Signed-off-by: Aaron Lowe <cyberflameu@gmail.com>

---------

Signed-off-by: CyberFlame <cyberflameu@gmail.com>
Signed-off-by: Aaron Lowe <cyberflameu@gmail.com>

CyberFlameGO committed:
[ci skip] update usage

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

renovate[bot] committed:
Update plugin fabric-loom to v1.8.10

renovate[bot] committed:
Update plugin fabric-loom to v1.8.11

renovate[bot] committed:
Update plugin fabric-loom to v1.8.12

renovate[bot] committed:
Update dependency gradle to v8.11 (#181)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
Update dependency gradle to v8.11.1 (#182)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
Update plugin fabric-loom to v1.8.13 (#184)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
Update plugin fabric-loom to v1.9.1 (#185)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
Update plugin fabric-loom to v1.9.2 (#186)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
Update dependency gradle to v8.12 (#187)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
Update dependency gradle to v8.12.1 (#188)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
Update plugin fabric-loom to v1.10.1 (#189)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

CyberFlameGO committed:
add 1.21.4

Signed-off-by: Aaron Lowe <cyberflameu@gmail.com>

CyberFlameGO committed:
[ci skip] remove weird moduled files

Signed-off-by: Aaron Lowe <cyberflameu@gmail.com>

renovate[bot] committed:
Update dependency gradle to v8.13 (#191)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
Update plugin fabric-loom to v1.10.2 (#192)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
Update plugin fabric-loom to v1.10.4 (#194)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
Update plugin fabric-loom to v1.10.5 (#195)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
Update dependency gradle to v8.14 (#198)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

CyberFlameGO committed:
1.21.5

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

renovate[bot] committed:
Update dependency gradle to v8.14.1 (#200)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
Update dependency gradle to v8.14.2 (#201)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
Update dependency gradle to v8.14.3 (#203)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
Update plugin fabric-loom to v1.11.4 (#204)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
Update plugin fabric-loom to v1.11.5 (#208)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
Update plugin fabric-loom to v1.11.7 (#211)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
Update plugin fabric-loom to v1.11.8 (#214)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

CyberFlameGO committed:
1.21.5 fix

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
1.21.8

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

renovate[bot] committed:
Update plugin fabric-loom to v1.12.1 (#216)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
Update Gradle to v9.2.0 (#224)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
Update plugin fabric-loom to v1.12.5 (#225)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
Update plugin fabric-loom to v1.13.3 (#226)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
Update plugin fabric-loom to v1.13.4 (#227)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
Update Gradle to v9.2.1 (#229)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
Update plugin fabric-loom to v1.13.6

renovate[bot] committed:
Update plugin fabric-loom to v1.14.1

renovate[bot] committed:
Update plugin fabric-loom to v1.14.3

renovate[bot] committed:
Update plugin fabric-loom to v1.14.4

renovate[bot] committed:
Update plugin fabric-loom to v1.14.5

renovate[bot] committed:
Update plugin fabric-loom to v1.14.6

renovate[bot] committed:
Update plugin fabric-loom to v1.14.7

renovate[bot] committed:
Update plugin fabric-loom to v1.14.8

renovate[bot] committed:
Update plugin fabric-loom to v1.14.10

renovate[bot] committed:
Update plugin fabric-loom to v1.15.1

renovate[bot] committed:
Update Gradle to v9.3.0

renovate[bot] committed:
Update plugin fabric-loom to v1.15.2

renovate[bot] committed:
Update plugin fabric-loom to v1.15.3

renovate[bot] committed:
Update Gradle to v9.3.1

renovate[bot] committed:
Update Gradle to v9.4.0

renovate[bot] committed:
Update plugin fabric-loom to v1.15.4 (#233)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
Update plugin fabric-loom to v1.15.5

renovate[bot] committed:
Update Gradle to v9.4.1

CyberFlameGO committed:
1.21.11 release

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
evil conflict

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
26.1

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

renovate[bot] committed:
Update plugin net.fabricmc.fabric-loom to v1.15.5

renovate[bot] committed:
Update gradle/actions action to v6 (#234)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
Update actions/checkout action to v6 (#230)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
Update actions/upload-artifact action to v7 (#222)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
Update github/codeql-action action to v4 (#219)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
Update actions/setup-java action to v5 (#213)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

CyberFlameGO committed:
Change Java version to 25 in build.yml

Updated Java version from 21 to 25 in build workflow.

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

CyberFlameGO committed:
Update Java version in release workflow to 25

Signed-off-by: CyberFlame <cyberflameu@gmail.com>

renovate[bot] committed:
Update plugin net.fabricmc.fabric-loom to v1.16.1

renovate[bot] committed:
Update Gradle to v9.5.0

renovate[bot] committed:
Update Gradle to v9.5.1

renovate[bot] committed:
Update plugin net.fabricmc.fabric-loom to v1.16.2

renovate[bot] committed:
Update plugin net.fabricmc.fabric-loom to v1.16.3

renovate[bot] committed:
Update plugin net.fabricmc.fabric-loom to v1.17.2

renovate[bot] committed:
Update plugin net.fabricmc.fabric-loom to v1.17.3

renovate[bot] committed:
Update plugin net.fabricmc.fabric-loom to v1.17.7

renovate[bot] committed:
Update plugin net.fabricmc.fabric-loom to v1.17.8

renovate[bot] committed:
Update plugin net.fabricmc.fabric-loom to v1.17.11

renovate[bot] committed:
Update Gradle to v9.6.0

renovate[bot] committed:
Update plugin net.fabricmc.fabric-loom to v1.17.12

renovate[bot] committed:
Update Gradle to v9.6.1

renovate[bot] committed:
Update plugin net.fabricmc.fabric-loom to v1.17.13

renovate[bot] committed:
Update plugin net.fabricmc.fabric-loom to v1.17.14

renovate[bot] committed:
Update plugin net.fabricmc.fabric-loom to v1.17.16
